Packs/RiskSense/Playbooks/DHS-CISA-FBI_Top_10_CVE_Exposure_-_RiskSense.yml

```diff
@@ -0,0 +1,460 @@
+id: DHS-CISA-FBI Top 10 CVE Exposure - RiskSense
+version: -1
+name: DHS-CISA-FBI Top 10 CVE Exposure - RiskSense
+description: Block IPs and apply the tag to assets that are vulnerable to DHS-CISA-FBI
+  Top 10 CVEs.
+starttaskid: '0'
+tasks:
+  '0':
+    id: '0'
+    taskid: 5d205ec0-77df-4b7f-869d-1c71c98da065
+    type: start
+    task:
+      id: 5d205ec0-77df-4b7f-869d-1c71c98da065
+      version: -1
+      name: ''
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '1'
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '1':
+    id: '1'
+    taskid: 1ef0f27b-376b-40b2-82b4-e70a93a11b30
+    type: condition
+    task:
+      id: 1ef0f27b-376b-40b2-82b4-e70a93a11b30
+      version: -1
+      name: Is RiskSense enabled ?
+      description: Returns 'yes' if integration brand is available. Otherwise returns
+        'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '10'
+      'yes':
+      - '13'
+    scriptarguments:
+      brandname:
+        simple: RiskSense
+    results:
+    - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '4':
+    id: '4'
+    taskid: 277b6725-ecde-4306-8d06-9ad8114fea5b
+    type: regular
+    task:
+      id: 277b6725-ecde-4306-8d06-9ad8114fea5b
+      version: -1
+      name: Block assets
```

Comment:
This is a deprecated command
Use this sub playbook instead
Block IP - Generic v2